packages/neuron-ui/src/services/remote/index.ts

```diff
@@ -40,6 +40,14 @@ export const showOpenDialog = (options: { title: string; message?: string }) =>
   return window.remote.require('electron').dialog.showOpenDialog(options)
 }
 
+export const showOpenDialogModal = (options: { title: string; message?: string }) => {
```

Comment:
What's the difference between `showOpenDialogModal` and `showOpenDialog`?

Reply:
The modal override passes in the current window. On macOS open file dialog usually appears as a modal sub window.